packages/-ember-data/tests/integration/record-data/record-data-test.ts

```diff
@@ -323,4 +367,294 @@ module('integration/record-data - Custom RecordData Implementations', function (
     assert.equal(calledGet, 3, 'called getAttr after notifyPropertyChange');
     assert.deepEqual(person.changedAttributes(), { name: ['old', 'new'] }, 'changed attributes passes through RD value');
   });
+
+  test("Record Data controls belongsTo notifications", async function (assert) {
+    assert.expect(6);
+    let called = 0;
+    let createCalled = 0;
+
+    let { owner } = this;
+    let belongsToReturnValue = { data: { id: '1', type: 'person' } };
+
+    class RelationshipRecordData extends TestRecordData {
+      constructor(storeWrapper) {
+        super();
+        this._storeWrapper = storeWrapper;
+      }
+
+      getBelongsTo(key: string) {
+        assert.equal(key, 'landlord', 'Passed correct key to getBelongsTo');
+        return belongsToReturnValue;
+      }
+
+      setDirtyBelongsTo(key: string, recordData: this | null) {
```

Comment:
did we change from `setBelongsTo` to `setDirtyBelongsTo`?

Reply:
Not sure, will investigate

Reply:
We did

Reply:
seems bad?  this is public api?  what was the reason to make this change?